### DISCUSSION TITLE
Permission system: Host should be able to give immunity to certain server settings (no wiring, karma, etc)

### DISCUSSION THREAD

**Discussion by CapoFantasma97** [FakeFishGames/Barotrauma#5338] [Ideas and suggestions]
Server settings are very limiting, as it's not possible to make **permissions on a per-player basis**.

The biggest **griefing** issue resides in **rewiring and removing wires**. Ideally, you should be able to give the **power of wiring only to trusted people**, if you want to; currently, it's only a _"yes everyone"_ or _"no everyone"_ switch, which either allows griefing to happen or greatly hinders creativity and customization of the ship.

Good security guards shouldn't be affected by karma either. If the host were able to disable karma only on trusted guards, they wouldn't incur penalties for doing their job.

This system could be integrated with the whitelist (therefore using the whitelist for permissions, rather than just access, when the "use whitelist" option is turned off), but it should be reworked to be based on Steam accounts, not the username and IP address.

**Comment by Ek-Videogames** [comment]
I agree with your points about security and medical not being penalized for doing their job
